### PR TITLE
Azure EventGrid: .NET SDK for 2019-02-01-preview API Version

### PR DESCRIPTION
**Pull request by ahamad-MS**
Add new .NET SDK generated files for the new 2019-02-01-preview API version which mainly has the following new features:
1. Pagination and search filtering for various List CRUD operations.
2. Manual create/delete for domain topics.
3. Add ServiceBus queue as destination.
4. Use null for IncludedEventTypes when event subscription is created to get all default event types.

We also add new test cases for the new features

### CODE REVIEW

**Comment by shahabhijeet** [comment]
@ahamad-MS you can see failing tests by clicking "default" job link and under articacts you can see your test results EventGrid*.trx

**Comment by ahamad-MS** [comment]
@shahabhijeet   All checks have passed. Can you please review and merge.

**Comment by shahabhijeet** [comment]
@AlexGhiondea can you review and merge the PR

**Comment by ahamad-MS** [comment]
@AlexGhiondea Can you please help review and merge the PR.

**Comment by shahabhijeet** [comment]
@weshaggard can you approve this PR, seems Alex is oof

File: src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid/Customization/Models/ResourceCredentials.cs [hunk 1]
```diff
@@ -9,11 +9,11 @@
 
 namespace Microsoft.Azure.EventGrid.Models
 {
-    public class TopicCredentials : ServiceClientCredentials
+    public class ResourceCredentials : ServiceClientCredentials
```
Comment (weshaggard) [inline src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid/Customization/Models/ResourceCredentials.cs]: Isn't this an API breaking change?

Comment (ahamad-MS) [reply src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid/Customization/Models/ResourceCredentials.cs]: I just realized that now. Good catch. I did not want to have duplicated code but will need to here.

**Review by kalyanaj — COMMENTED** [review]
The version/release notes in the CSPROJ file(s) need to be updated as well.

**Comment by ahamad-MS** [comment]
Hello @weshaggard  .. All checks are passed and all comments are taken care of. Kindly take a look. thanks

**Review by weshaggard — APPROVED** [review]
I don't have all the context but from a high level the changes look reasonable to me.

**Pull request merged**